Pipfile

```diff
@@ -17,6 +17,7 @@ djangorestframework = "*"
 mysqlclient = "*"
 sentry-sdk = "*"
 django = "*"
+django-cors-middleware = "*"
```

Comment:
did you add CORS to the list of middlewares too?

Reply:
Thanks, missed that! Fixed